src/eatery_db/campus_eatery.py

```diff
@@ -244,7 +256,7 @@ def parse_static_op_hours(data_json, eatery_model):
                         )
                     if not new_events:
                         new_operating_hours.append(
-                            (CampusEateryHour(eatery_id=eatery_model.id, date=new_date.isoformat(),), dining_items)
+                            (CampusEateryHour(eatery_id=eatery_model.id, date=new_date.isoformat()), dining_items)
```

Comment:
hm are you sure this doesn't break anything

Reply:
It's just an extra comma in a constructor, right? Can that affect something?

Reply:
just checking to make sure you know for sure it works, a comma sometimes could make or break things :((